extensions/notebook/src/test/common/notebookUtils.test.ts

```diff
@@ -50,12 +50,12 @@ describe('notebookUtils Tests', function (): void {
 			should(azdata.nb.notebookDocuments.length).equal(0, 'There should be not any open Notebook documents');
 			await notebookUtils.newNotebook(undefined);
 			should(azdata.nb.notebookDocuments.length).equal(1, 'There should be exactly 1 open Notebook document');
-			should(azdata.nb.notebookDocuments[0].fileName).equal('Notebook-0', 'The first Untitled Notebook should have an index of 0');
+			should(azdata.nb.notebookDocuments[0].fileName).equal('Notebook-1', 'The first Untitled Notebook should have an index of 1');
```

Comment:
Untitled Text and Query document names start counting up from 1, so I updated notebooks to have the same naming behavior.